src/shared/store/ApiStore/ApiStore.ts

```diff
@@ -0,0 +1,51 @@
+import qs from "qs";
+
+import {
+  ApiResponse,
+  IApiStore,
+  RequestParams,
+  StatusHTTP,
+  HTTPMethod,
+} from "./types";
+
+export default class ApiStore implements IApiStore {
+
+    readonly baseUrl: string;
+
+    constructor(baseUrl: string) {
+        this.baseUrl = baseUrl;
+    }
+    
+    getconsole(str: string): string {
+        throw new Error("Method not implemented.");
+    }
+
+    request<SuccessT, ErrorT = any, ReqT = {}>(
+        params: RequestParams<ReqT>
+      ): Promise<ApiResponse<SuccessT, ErrorT>> {
+          try {
+              let urlForRequest: string;
+              if (params.method === HTTPMethod.GET && params.data) {
+                  const resolveAddress: string = "?" + qs.stringify(params.data);
```

Comment:
лучше везед использовать шаблонные строки 
```suggestion
                  const resolveAddress: string = `?${qs.stringify(params.data)}`;
```

Reply:
done